strimzi/strimzi-kafka-operator
# [MO] - [dyn.conf] -> system tests

Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### Type of change

- Enhancement / new feature


### Description

This PR adding tests for dynamic configuration.

#### How do these tests work?

Basically each property is verified by method `verifyDynamicConfiguration` from `KafkaUtils`. Inside `verifyDynamicConfiguration` we have two major steps. The first one is to update the selected property and then we call the method `verifyThatRunningPodsAreStable` from `PodUtils` to make sure that pods did not roll (dynamic conf). The second one is verified phase when I just compare 2 values.

#### How long?

for suite `DynamicConfigurationSharedST` :
<img width="950" alt="image" src="https://user-images.githubusercontent.com/30839163/91579053-59679100-e94b-11ea-842a-b911a744c511.png">


### Checklist
